### PR TITLE
Hackage DB logic from #6

### PR DESCRIPTION
**Pull request by angerman**

### CODE REVIEW

**Comment by angerman** [comment]
CI failed, because we added hackage-db. We'll need to regenerate the nix expression. However the haskell.nix/compat/driver likely won't understand the new hackage expressions.

**Comment by angerman** [comment]
With the recent changes to angerman/hackage.nix#2 and angerman/haskell.nix#2, this now allows us to run
```
$ nix build -f default.nix hsPkgs.nix-tools.components.exes.cabal-to-nix
```
and obtain `result/bin/cabal-to-nix`.

File: nix/cabal-licenses.nix [hunk 1]
```diff
@@ -1,2 +1,5 @@
 stdenv: with stdenv.lib.licenses;
-{ BSD-3-Clause = bsd3; }
+{ BSD-3-Clause = bsd3;
+  BSD-2-Clause = bsd2;
+  MIT = mit;
+}
```
Comment (ElvishJerricco) [inline nix/cabal-licenses.nix]: @angerman Should this change be dropped, so as not to conflict with #11?

Comment (angerman) [reply nix/cabal-licenses.nix]: yep. this is dead :-)

Comment (angerman) [reply nix/cabal-licenses.nix]: I'll rebase #11, after this to clean up.

**Comment by angerman** [comment]
Alight, cachix and travis don't play along well right now, builds successfully on macOS and nixOS.

**Pull request merged**